components/dialog/test/dialog.visual-diff.js

```diff
@@ -44,34 +44,40 @@ describe('d2l-dialog', () => {
 					});
 
 					it('opened', async function() {
-						await helper.open(page, '#dialog');
```

Comment:
The helper simply waits for the `d2l-dialog-open` event, which technically is probably the safest because it ensure that the dialog has been sized and the focus applied.  So in this case I would tend to prefer waiting for the event, and unfortunately I'd suggest app devs do the same.  They should however, not need to wait an arbitrary time anymore.

Reply:
Yeah seems like waiting for the event is best, since it's part of the dialog's public API anyway.